lib/Backend/InliningDecider.cpp

```diff
@@ -624,26 +624,8 @@ bool InliningDecider::GetBuiltInInfoCommon(
         break;
 #endif
 
-#ifdef ENABLE_SIMDJS
-    // SIMD_JS
-    // we only inline, and hence type-spec on IA
-#if defined(_M_X64) || defined(_M_IX86)
     default:
-    {
-#if 0 // TODO OOP JIT, inline SIMD
-        // inline only if simdjs and simd128 type-spec is enabled.
```

Comment:
The code was already behind `#if 0` and I confirmed with @MikeHolman that this is not something we will need in the future.
I don't see why we shouldn't cleanup the code.